docs/src/content/components/button/Demos/FlatDemo.js

```diff
@@ -8,7 +8,7 @@ export const code = `<View style={{flexDirection: 'row', alignItems: 'center', j
   <Button text={'Icon'} type="flat" color={'#E91E63'} icon={<Icon name="favorite" />} />
   <Button text={'Radius'} type="flat" color={'#FF5722'} radius={20} />
   <Button text={'Dense'} type="flat" color={'#673AB7'} dense />
-  <Button text={'Loading'} type="flat" color={'#009688'} loading />
```

Comment:
We need to keep the text here as the use case is a button with text transitions into a progress button